code/modules/power/apc.dm

```diff
@@ -931,6 +933,14 @@
 		update_icon()
 		update()
 
+	else if (href_list["emergency_lighting"])
+		emergency_lights = !emergency_lights
+		for(var/obj/machinery/light/L in area)
+			if(!initial(L.no_emergency)) //If there was an override set on creation, keep that override
+				L.no_emergency = emergency_lights
+				INVOKE_ASYNC(L, /obj/machinery/light/.proc/update, FALSE)

```

Comment:
`.proc`?

Reply:
That's right.  It's a proc typepath shortcut, there's a comment about them at the top of callback.dm